.vscode/tex_snippets.code-snippets

```diff
@@ -9,19 +9,27 @@
     "Abkürzung": {
         "prefix": ["abk", "dhgeabk"], // auf welchem namen der snippet auffindbar ist
         "scope": "latex,tex", // welche files der snippet betreffen soll
-        "body": ["\\DeclareAcronym{${1:ac-used-name}}{",
-                "   short = {${2:abk}},",
-                "   long = {${3:non_abk}}",
-                "}"
+        "body": ["\\newacronym{${1:abk-id}}",
+                "    {${2:abk}}",
+                "    {${3:non_abk}}"

```

Comment:
Hätte das evtl. sogar vollständig entfernt, da mehr als Singleline nicht mehr nötig ist. \
Es sollte also wahrscheinlich aus meiner Sicht nur noch eine abk Snippet geben.

Reply:
Also ich hatte das extra mit reingenommen, weil ich in PA4 zb einen Abk-Eintrag und einen Glossar-Eintrag für, halt dich fest, die App-Verbundmetadaten-URL gebraucht habe. Also einmal `ac:avu` und einmal `gls:avu`

Das wäre dann einheitlich zu den Präfixen `fig:` und `tab:`

Reply:
Hmm... geht mir halt mehr darum das es wie zuvor ist, mir wäre es egal insofern @RvNovae auch für die Prefixe wäre^^.

Reply:
Ja, da ist auch was daran...

Am Ende läuft es glaube ich auf zwei Gedanken hinaus: wollen wir...

- Anfänger an die Hand nehmen und Ordnung vorgeben oder...
- dem Nutzer maximale Freiheit geben, dann muss er/sie halt selbst Ordnung halten --> in dem Falle Doku sinnvoll